scripts/jenkins/Jenkinsfile.daily

```diff
@@ -80,13 +84,42 @@ def cleanup_docker_images(target_node) {
     }]
 }
 
-def generate_file_cleanup_stages() {
+def generateDockerRepoMaintanenceStages() {
+    def nodes = nodesByLabel label:"docker&&registry"
+    def tasks = [:]
+
+    nodes.each { target_node ->
```

Comment:
Why target_node not camel case?

Reply:
It does not really match with your description of the PR: lowerCamelCaseEverywhere

And I suggest to actually do that, mixing CamelCase and other_ways is not a good idea. When someone else starts modifying the code, inconsistencies will appear.